autogen/agentchat/groupchat.py

```diff
@@ -702,7 +746,14 @@ def validate_speaker_name(recipient, messages, sender, config) -> Tuple[bool, Un
         # Two-agent chat for speaker selection
 
         # Agent for checking the response from the speaker_select_agent
-        checking_agent = ConversableAgent("checking_agent", default_auto_reply=max_attempts)
+        checking_agent = ConversableAgent(
```

Comment:
The checking agent does not use llm_config. @marklysze can correct me if I'm wrong.

Reply:
Hi, I think that's correct. I'll try to verify and clean up the code today

Reply:
> The checking agent does not use llm_config. @marklysze can correct me if I'm wrong.

Yes, all responses handled through local functions.

Reply:
Fixed with cf72ebf4a3bb5bb08a71a2200c207ba94c8d737e